### ISSUE TITLE
[Feature] Add CTI/Aperture Adjustments for EXP CVBS and S-Video Inputs

### ISSUE THREAD

**Issue by GK6475** [marqs85/ossc_pro#19]
In scaler and line multiplier mode, with no filters applied, the image from the Legacy AV in can be a bit too sharp, and has some "ringing" around color borders and object edges. It would be great to have a way to adjust this feature, given that some games/consoles will look way better with less of it. The RT5X has CTI and Aperture settings which seem to adjust this sort of thing in different aspects of the image. The Legacy AV in would benefit greatly from having some way to configure the sharpness/ringing to the user's preference.

**Comment by marqs85** [comment]
Some CTI settings exist in the video decoder chip, perhaps they can be exposed in the next firmware.

**Comment by GK6475** [comment]
Yeah, that'd be great. It would certainly make a difference on consoles that have plenty of dot crawl, such as the N64.

**Comment by marqs85** [comment]
CTI options were added in 97ba1931642764354c842c019f9e1d786606f210.
